### PR TITLE
feat: Implement help command

### PR DESCRIPTION
**Pull request by benwoo1110**
Images of how help command looks:
image

image

image

image

Test build:
[Multiverse-Core-5.0.0-SNAPSHOT.zip](https://github.com/Multiverse/Multiverse-Core/files/10773505/Multiverse-Core-5.0.0-SNAPSHOT.zip)

### CODE REVIEW

File: src/main/java/com/onarandombox/MultiverseCore/commands/UsageCommand.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    @HelpCommand
+    @Subcommand("help")
+    @CommandPermission("multiverse.core.help")
+    @CommandCompletion("@commands:mv")
+    @Syntax("[filter] [page]")
+    @Description("Show Multiverse-Core Command usage.")
+    public void onUsageCommand(CommandHelp help) {
+        if (help.getIssuer().isPlayer()) {
+            // Prevent flooding the chat
+            help.setPerPage(4);
```
Comment (zax71) [inline src/main/java/com/onarandombox/MultiverseCore/commands/UsageCommand.java:29]: Maybe a `--all` flag to not show pages?

File: src/main/java/com/onarandombox/MultiverseCore/commands/UsageCommand.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+@CommandAlias("mv")
+public class UsageCommand extends MultiverseCoreCommand {
+    public UsageCommand(@NotNull MultiverseCore plugin) {
+        super(plugin);
+    }
+
+    @HelpCommand
+    @Subcommand("help")
+    @CommandPermission("multiverse.core.help")
+    @CommandCompletion("@commands:mv")
```
Comment (zax71) [inline src/main/java/com/onarandombox/MultiverseCore/commands/UsageCommand.java:23]: Maybe add  `@range|@commands:mv` here to show that you can specify a page

**Review by zax71 — APPROVED** [review]
Not much this time

**Review by dumptruckman — APPROVED** [review]

File: src/main/java/com/onarandombox/MultiverseCore/commands/UsageCommand.java (hunk 2 above)
Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/commands/UsageCommand.java:23]: from experience, typing a single number on the keyboard is easier than tabbing through a range of numbers

File: src/main/java/com/onarandombox/MultiverseCore/commands/UsageCommand.java (hunk 1 above)
Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/commands/UsageCommand.java:29]: hmm yea, possible

Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/commands/UsageCommand.java:29]: looking at how CommandHelp context is parsed, I will hold off on this idea for now

**Pull request merged**